src/Storage.php

```diff
@@ -193,7 +193,8 @@ public function addAssignment(string $userId, Item $item): void
     public function assignmentExist(string $name): bool
     {
         foreach ($this->getAssignments() as $assignmentInfo) {
-            foreach ($assignmentInfo as $itemName => $assignment) {
+            $itemNames = array_keys($assignmentInfo);
+            foreach ($itemNames as $itemName) {
```

Comment:
Originally we were iterating over keys without creating another array with keys only. New code is less efficient.